compiler/plugins/target/AMD-AIE/aie/AMDAIECreatePathFindFlows.cpp

```diff
@@ -38,16 +38,17 @@ using xilinx::AIE::TileOp;
 
 namespace mlir::iree_compiler::AMDAIE {
 
-TileOp getOrCreateTile(OpBuilder &builder, DeviceOp &device, int col, int row) {
+static TileOp getOrCreateTile(OpBuilder &builder, DeviceOp &device, int col,
```

Comment:
```suggestion
static TileOp getOrCreateTile(OpBuilder &builder, DeviceOp device, int col,
```
Nit: ops are usually passed by value as they are really just like pointers.